sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -258,9 +256,9 @@ multilib_src_configure() {
 		-Drootlibdir="${EPREFIX}/usr/$(get_libdir)"
 		# Avoid infinite exec recursion, bug 642724
 		-Dtelinit-path="${EPREFIX}/lib/sysvinit/telinit"
-		# no deps
 		-Dima=true
-		-Ddefault-hierarchy=$(usex cgroup-hybrid hybrid unified)
+		# Flatcar: Use unified hierarchy now that docker-20.10 is available
+		-Ddefault-hierarchy=unified
```

Comment:
I think we could drop this diff if we can make sure that `cgroup-hybrid` use flag is disabled.